SEED-platform/seed
# Filter cycle specific labels on inventory list

#### Any background context you want to provide?
The new Inventory List page's filter auto-complete shows all labels that are associated with any property, in any cycle. This differs from the functionality on Inventory List (legacy) page where only labels associated with the cycle's inventory are present.

#### What's this PR do?
- Adds cycle-specific inventory filters to the returned labels. The prior functionality returned all labels that have been applied to a property regardless of cycle.
- Changes the order of function calls. For context, the filter labels function generated labels and then found the cycle's inventory. However, to accurately filter labels the inventory must first exist. 

#### How should this be manually tested?
- create at least 2 cycles. 
- apply cycle specific labels to a few properties
- apply cross-cycle labels to a few properties
- click the "Filter by Label" input field. The auto-complete should only include labels applied to this cycle.

#### What are the relevant tickets?


#### Screenshots (if appropriate)
2 cycles with unique and common labels.
![Screenshot 2023-02-13 at 3 02 03 PM](https://user-images.githubusercontent.com/58446472/218584676-84b35ef3-f026-4dc9-9fcf-c13ca7be575f.png)
![Screenshot 2023-02-13 at 3 00 55 PM](https://user-images.githubusercontent.com/58446472/218584679-d2e307de-8a49-4700-8766-96f97d9b2e94.png)
